build.sh

```diff
@@ -36,32 +30,30 @@ function build {
     gpart add -t freebsd-boot -s 1024 ${md_dev}
     gpart bootcode -b /boot/pmbr -p ${gptboot} -i 1 ${md_dev}
     gpart add -t efi -s 128M ${md_dev}
-    gpart add -s 1G -l swapfs -t freebsd-swap ${md_dev}
```

Comment:
You don't create any swap partition?

Reply:
No, what for? Especially with such a laughable size...